src/python/README.md

```diff
@@ -8,7 +8,7 @@ Beta : Core behavior well-used and proven; bugs lurk off the beaten path.
 
 PREREQUISITES
 -------------
-- Python 2.7, virtualenv, pip
+- virtualenv, pip, python-dev, python3-dev
```

Comment:
Since this is under `src` I'm considering the audience of these instructions to be developers potentially contributing to gRPC Python, not just users (of course, gRPC's users *are* developers, but by "users" I mean "developers who merely want to develop on top of gRPC rather than potentially change it..."). So for that audience, we absolutely want them installing both the 2 and 3 families of packages and running their changes against both 2 and 3 (as soon as we support both).